### ISSUE TITLE
Mission planner 1.3.35 Auto Analysis

### ISSUE THREAD

**Issue by JeyPi** [diydrones/MissionPlanner#1127]
When I try to auto analyse a log (arduplane 3.5.0 Pixhawk) I get a bad input message.
Win. XP7
[Log](https://www.dropbox.com/s/nqmjqglkhesjsmq/1.BIN?dl=0)

**Comment by stik79** [comment]
Same thing Win10. Attached log.
NpyAccessLib.dll exists in DLL directory within MissionPlanner.

[NpyError.txt](https://github.com/diydrones/MissionPlanner/files/119986/NpyError.txt)

**Comment by stik79** [comment]
As a quick workaround launching missionplanner from within the DLLs directory (i.e. ..\MissionPlanner.exe) picks numpy up and works - which is strange because if I do a print sys.path it has added the DLLs directory running normally (i.e. from the application root) too...

Unfortunately I'm not enough of a python/.Net guru to diagnose much more :(

**Comment by RestlessC0bra** [comment]
Same thing here. Win8.1 Gives me error messages

The type initializer for 'NumpyDotNet.NpyCoreApi' threw an exception

and then

Bad input file

I can't analyze any logs!

**Comment by Hankster1228** [comment]
Same thing here. 

 Windows 7 gives me the same error message when I try to auto analyze a log.  I have the most recent version of Mission Planner 1.3.35 build 1.1.5878.12941 downloaded 3 pm central February 11, 2016.  Error message is: The type initializer for 'NumpyDotNet.NpyCoreApi' threw an exception followed by a small dialog box stating "Bad Input File".  The file can be downloaded and analyzed but just not auto analyzed.

**Comment by meee1** [comment]
as a temp fix, please delete NumpyDotNet.dll from the MP folder.

it should remain in the "dlls" folder however.

**Comment by Hankster1228** [comment]
Thanks! That fix worked.

On Sat, Feb 13, 2016 at 2:25 AM, Michael Oborne <notifications@github.com>
wrote:

> as a temp fix, please delete NumpyDotNet.dll from the MP folder.
>
> it should remain in the "dlls" folder however.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/MissionPlanner/issues/1127#issuecomment-183624756>
> .
>



-- 
Hank Hamner

**Comment by JeyPi** [comment]
Well not for me. After that I get first -  **System Exit** message and than **Bad input file**

**Comment by Hankster1228** [comment]
I went to my Mission Planner program folder (*c:\program files
(x86)\Mission Planner\*) and renamed the NumpyDotNet.dll to NumpyDotNet.DAL
and it fixed the problem.  I renamed the DLL rather that erasing it just in
case I screwed up and deleted the wrong dll.  I'm running Windows 7.

Good luck!

On Sun, Feb 14, 2016 at 12:34 AM, JeyPi <notifications@github.com> wrote:

> Well not for me. After that I get first - *System Exit* message and than *Bad
> input file*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/MissionPlanner/issues/1127#issuecomment-183831449>
> .
>



-- 
Hank Hamner
